openapi/src/main/java/io/helidon/openapi/OpenAPISupport.java

```diff
@@ -630,24 +656,35 @@ static SEOpenAPISupportBuilder builderSE() {
      * Helidon SE apps and once for use from the Helidon MP-provided OpenAPI
      * service. This lets us constrain what use cases are possible from each
      * (for example, no anno processing from SE).
+     *
+     * @param <T> concrete subclass of OpenAPISupport
+     * @param <B> concrete subclass of OpenAPISupport.Builder
      */
-    public abstract static class Builder implements io.helidon.common.Builder<OpenAPISupport> {
+    public abstract static class Builder<T extends OpenAPISupport<T, B>, B extends Builder<T, B>>
+            implements io.helidon.common.Builder<OpenAPISupport> {
```

Comment:
Should be `implements io.helidon.common.Builder<T>`

Reply:
Fixed.